shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -33,21 +45,42 @@ class PlatformViewIOS final : public PlatformView {
 
   ~PlatformViewIOS() override;
 
+  /**
+   * The `PlatformMessageRouter` is the iOS bridge connecting the shell's
+   * platform agnostic `PlatformMessage` to iOS's channel message handler.
+   */
   PlatformMessageRouter& GetPlatformMessageRouter();
 
+  /**
+   * Returns the `FlutterViewController` currently attached to the `FlutterEngine` owning
+   * this PlatformViewIOS.
+   */
   fml::WeakPtr<FlutterViewController> GetOwnerViewController() const;
+
+  /**
+   * Updates the `FlutterViewController` currently attached to the `FlutterEngine` owning
+   * this PlatformViewIOS.
+   */
   void SetOwnerViewController(fml::WeakPtr<FlutterViewController> owner_controller);
+
+  /**
+   * Called one time per `FlutterViewController` when the `FlutterViewController`'s
+   * UIView is first loaded.
+   *
+   * Can be used to perform late initialization after `FlutterViewController`'s
+   * init.
+   */
   void attachView();
 
+  /**
+   * Called through when an external texture such as video or camera is
+   * given to the `FlutterEngine` or `FlutterViewController`.
```

Comment:
These two docstrings talk about when a method is called.  Docstrings should explain what the procedure does, not when it is called.

Reply:
ya, this is unfortunately same as above. It's not a "docstring" since the whole thing is private. It's code comment rather since this existing code (which was here before FlutterEngine etc existed) already have expectations on when it needs to be called. What it offers to its "consumer"'s perspective is already opaque.